### PR TITLE
feat: bot to check for failed releases

### PR DESCRIPTION
**Pull request by bcoe**
There have been a few cases where we merge a release PR and a piece of machinery somewhere in the stack fails to deploy a module, this bot will open an issue when this happens.

For now, let's land out a stubbed out version and make sure that crons work.

### CODE REVIEW

File: packages/failurechecker/cron [hunk 1]
```diff
@@ -0,0 +1 @@
+*/5 * * * *
```
Comment (alexander-fenster) [inline packages/failurechecker/cron]: no `\n`

File: packages/failurechecker/src/app.ts [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { GCFBootstrapper } from 'gcf-utils';
+import appFn from './failurechecker';
+
+const bootstrap = new GCFBootstrapper();
+module.exports.failurechecker = bootstrap.gcf(appFn);
```
Comment (alexander-fenster) [inline packages/failurechecker/src/app.ts]:
```ts
const failurechecker = bootstrap.gcf(appFn);
export {failurechecker};
```

File: packages/failurechecker/src/index.d.ts [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+ * limitations under the License.
+ */
+
+// define types for a few modules used by probot that do not have their
+// own definitions published. Before taking this step, folks should first
+// check whether type bindings are already published.
+declare module "promise-events" {
+  class EventEmitter {
+  }
+}
```
Comment (alexander-fenster) [inline packages/failurechecker/src/index.d.ts]: `\n`

File: packages/failurechecker/src/failurechecker.ts [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+export = (app: Application) => {
+  app.on(
+    ['issues.opened', 'pull_request.opened', 'schedule.repository'],
+    async context => {
+      const config = (await context.config(
+        CONFIGURATION_FILE_PATH,
+        {}
+      )) as Configuration;
+
+      app.log("it's alive!");
```
Comment (alexander-fenster) [inline packages/failurechecker/src/failurechecker.ts]: do we need it? :)

File: packages/failurechecker/test/fixtures/config/valid-config.yml [hunk 5]
```diff
@@ -0,0 +1 @@
+randomBoolean: true
```
Comment (alexander-fenster) [inline packages/failurechecker/test/fixtures/config/valid-config.yml]: sorry!!! but `\n`

File: packages/failurechecker/tsconfig.json [hunk 6]
```diff
@@ -0,0 +1,15 @@
...
+    "outDir": "build"
+  },
+  "include": [
+    "src/*.ts",
+    "src/**/*.ts",
+    "test/*.ts",
+    "test/**/*.ts",
+    "system-test/*.ts"
+  ]
+}
```
Comment (alexander-fenster) [inline packages/failurechecker/tsconfig.json]: and final `\n`!

File: packages/failurechecker/src/failurechecker.ts (hunk 4 above)
Comment (bcoe) [reply packages/failurechecker/src/failurechecker.ts]: This will be more exciting if the cron is working :stuck_out_tongue:

**Pull request merged**